css/index.css

```diff
@@ -174,6 +174,57 @@ footer {
     margin: 40px 0 20px;
 }
 
+@media(max-width: 768px){
```

Comment:
Comments are your friend! It may help others / you to comment the kind of device you are targeting with this media query.